### PR TITLE
Sample Table TSV Download

### PR DESCRIPTION
**Pull request by lvreynoso**
### Description

What you see:

Screenshot from 2021-05-11 15-37-34

What you get:

Screenshot from 2021-05-11 15-37-52

#### Issue
[ch127256](https://app.clubhouse.io/genepi/story/127256)

### Test plan

Go to sample table, see button, download tsv, open up tsv

### CODE REVIEW

File: src/frontend/src/common/components/library/data_subview/index.tsx [hunk 1]
```diff
@@ -68,41 +97,30 @@ const DataSubview: FunctionComponent<Props> = ({
...
-    })
-
-    return [tsvHeaders, tsvData]
-  }
-
   const render = (tableData: TableItem[]) => {
-    console.log(tableData)
-    const [tsvHeaders, tsvData] = tsvDataMap(tableData)
-    const separator = "\t"
+    console.log(tableData);
```
Comment (tihuan) [inline src/frontend/src/common/components/library/data_subview/index.tsx:101]: rogue console.log 😆 !

**Review by tihuan — APPROVED** [review]
NIICE!! Looks super good! Just one console.log to remove, otherwise LGTM 🚀 🚀 🚀 

Thanks so much for adding this, @lvreynoso 🙏 ✨

File: src/frontend/src/common/components/library/data_subview/index.tsx (hunk 1 above)
Comment (lvreynoso) [reply src/frontend/src/common/components/library/data_subview/index.tsx:101]: odd that linting didn't catch this

Comment (tihuan) [reply src/frontend/src/common/components/library/data_subview/index.tsx:101]: Ooh didn't notice that lol Adding that in this [PR](https://github.com/chanzuckerberg/aspen/compare/thuang-eslint-no-console?expand=1) PTAL thank you!

**Pull request merged**